shared/settings/files/index.js

```diff
@@ -1,28 +1,110 @@
 // @flow
 import * as React from 'react'
 import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
 import * as Kb from '../../common-adapters'
-import {fileUIName, isLinux} from '../../constants/platform'
+import {InlineDropdown} from '../../common-adapters/dropdown'
+import * as Platform from '../../constants/platform'
 import * as Styles from '../../styles'
 import SystemFileManagerIntegrationBanner from '../../fs/banner/system-file-manager-integration-banner/container'
 import RefreshDriverStatusOnMount from '../../fs/common/refresh-driver-status-on-mount'
+import RefreshSettings from './refresh-settings'
+import flags from '../../util/feature-flags'
 
 type Props = {|
+  areSettingsLoading: boolean,
   driverStatus: Types.DriverStatus,
-  onEnable: () => void,
   onDisable: () => void,
+  onEnable: () => void,
   onShowKextPermissionPopup: () => void,
+  spaceAvailableNotificationThreshold: number,
+  onChangedSyncNotifications: (number) => void,
+  onEnableSyncNotifications: () => void,
+  onDisableSyncNotifications: () => void,
 |}
 
+export const allowedNotificationThresholds = [
+  100 * 1024 ** 2,
+  1024 ** 3,
+  10 * 1024 ** 3,
```

Comment:
maybe add one more option in like the 2-5 GB range? I feel like I'd want to set it somewhere around there personally.